src/components/ui/DropdownMenuUsageExample.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+/**
+ * This file is used as an example for the Dropdown components.
+ * Once we're done with the example, we can delete this file.
+ */
+import { DeleteIcon, MenuKebabIcon, PenIcon } from '@/assets/images/icons';
+import { ContainerCount } from '@/features/groups/components/GroupCardList/ContainerCount';
+import { UserCount } from '@/features/groups/components/GroupCardList/UserCount';
+import { cn } from '@/lib/tailwind/utils';
+
+import {
+  Button,
+  Card,
+  DropdownMenu,
+  DropdownMenuButton,
+  DropdownMenuButtonIcon,
+  DropdownMenuButtonText,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+} from '.';
+
+export const DropdownMenuUsageExample = () => {
+  return (
+    <div className="p-4 flex flex-col gap-3">
+      {Array.from({ length: 10 }).map((_, i) => (
+        <DropDownMenuWithCardExample key={i} />
+      ))}
+    </div>
+  );
+};
+
+const DropDownMenuWithCardExample = () => {
+  const handleRenameClick = () => {
+    alert('Rename clicked!');
+  };
+
+  const handleDeleteClick = () => {
+    alert('Delete clicked!');
+  };
+  return (
+    <Card asChild>
+      <div className="flex items-center justify-between pl-4 py-2 pr-0">
```

Comment:
The approach of styling the card and its content appears to need to be revised. If you don't know how to improve, feel free to ask :)

- Overriding the paddings defined in the Card component should be avoided
- This definition `[calc(100%-3.5rem)]` is not an appropriate approach to determine the layout
- The `Button` element should take up the full width of the card.

Reply:
> After most of card component's design has changed, wouldn't it better not to set default padding to Card component in the first place?
> Considering that the most of the card component now has kebab-menu button on right and doesn't need padding on right of it, having default padding for whole card content seems like unnecessary. Without default padding, we don't have to add negative margin or consider other way to solve the problem...

I understand your points, and agree to remove the paddings defined in the `Card` component